text-serializer-gson/src/main/java/net/kyori/adventure/text/serializer/gson/GsonComponentSerializer.java

```diff
@@ -111,20 +110,21 @@ static Builder builder() {
    */
   @NotNull JsonElement serializeToTree(final @NotNull Component component);
 
+  /**
+   * Creates a new {@link Builder} from this {@link GsonComponentSerializer}.
+   *
+   * @return a new {@link Builder}
+   * @since 4.9.0
+   */
+  @Override
+  @NotNull GsonComponentSerializer.Builder toBuilder();
+
   /**
    * A builder for {@link GsonComponentSerializer}.
    *
    * @since 4.0.0
    */
-  interface Builder extends Buildable.Builder<GsonComponentSerializer> {
-    /**
-     * Sets that the serializer should downsample hex colors to named colors.
-     *
-     * @return this builder
-     * @since 4.0.0
-     */
-    @NotNull Builder downsampleColors();
-
+  interface Builder extends JsonComponentSerializer.Builder {
```

Comment:
I don't think we can change the type of the builder here. Methods need to return `GsonComponentSerializer` so we don't break anything. Perhaps it needs to extend _both_ interfaces for now, ensuring all the methods return the old method?

Reply:
Yeah I actually forgot that the return types are all screwed up. I can likely fix it by using a `<B extends Builder<B>>`, since the super builder shouldn't be used standalone anyway.

Reply:
Actually, I see why I didn't do this. Let me just override all of these methods instead.